channels/fr.m3u

```diff
@@ -255,6 +255,8 @@ https://live.creacast.com/mirabelletv/smil:mirabelletv.smil/playlist.m3u8
 https://streamer01.myvideoplace.tv/streamer02/hls/MDS_VIA_PAD_301117.m3u8
 #EXTINF:-1 tvg-id="W9.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/7FFFvCN.jpg" group-title="Entertainment",W9 (720p)
 http://flusonic-1.platinum-tv.com/W9/mono.m3u8?token=test
+#EXTINF:-1 tvg-id="WéoNordPasdeCalais.fr" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="Local",Wéo (Nord-Pas-de-Calais) (480p) [Not 24/7]
+https://iptv.dc-team.com/query?url=https://www.dailymotion.com/video/x62islg
```

Comment:
This one is the same than the "Wéo Hauts-de-France" link below.

Note that the Nord-Pas-de-Calais region is also inside the Hauts-de-France (Picardy, Nord-Pas-de-Calais).